pennylane/debugging.py

```diff
@@ -208,3 +226,82 @@ def breakpoint():
 
     debugger = PLDB(skip=["pennylane.*"])  # skip internals when stepping through trace
     debugger.set_trace(sys._getframe().f_back)  # pylint: disable=protected-access
+
+
+def state():
+    """Compute the state of the quantum circuit.
+
+    Returns:
+        Array(complex): Quantum state of the circuit.
+    """
+    with qml.queuing.QueuingManager.stop_recording():
+        m = qml.state()
+
+    return _measure(m)
+
+
+def expval(op):
+    """Compute the expectation value of an observable.
+
+    Args:
+        op (Operator): The observable to compute the expectation value for.
+
+    Returns:
+        complex: Quantum state of the circuit.
+    """
+
+    qml.queuing.QueuingManager.active_context().remove(op)  # ensure we didn't accidentally queue op
```

Comment:
Is this really needed? What happens if the op is not in the queue?

Reply:
If the op is not in the queue to begin with it does nothing, if it is in the queue, it is removed.